app/views/keywords/_link.html.slim

```diff
@@ -0,0 +1 @@
+li = link_to nil, link, target: :_blank
```

Comment:
this is tooo small for a view partial @gutakk we could go without this partial

Reply:
Fixed in 281edf98f83ecc02ee5cae9c6ab67494513883f9